src/packages/cli/src/args.ts

```diff
@@ -219,6 +221,12 @@ export default function (version: string, isDocker: boolean) {
             }
 
             return true;
+          })
+          .option("cli.detach", {
```

Comment:
I'm not sure on this name-spacing:
➕  It tidies up the logic for breaking out the namespaces
➕  With a default value, we can access `args.cli.detach` without having to check whether `cli === undefined`
➖  It adds an unnecessary namespace, which we are locked into maintaining
➖  It pollutes the help text

I'm also unsure about placement in the help text - presently it's at the bottom with "general" options, where I'm not sure if fits, but I'm also not keen to put in a separate "Cli" section, with _just_ detach.

Keen to hear people's opinions.

```
Server:
      --server.ws                        Enable a websocket server.                            [boolean] [default: true]
      --server.wsBinary                  Whether or not websockets should response with binary data (ArrayBuffers) or
                                         strings.                     [choices: "true", "false", "auto"] [default: auto]
      --server.rpcEndpoint               Defines the endpoint route the HTTP and WebSocket servers will listen on.
                                                                         [default: "/" (Ethereum), "/rpc/v0" (Filecoin)]
      --server.chunkSize                 For memory and performance reasons ganache may respond with chunked
                                         transfer-encoding over HTTP and fragmented send over WebSockets. This option
                                         allows you to control the approximate size of each chunk.
                                                                                             [number] [default: 1048576]
  -h, --server.host, --host, --hostname  Hostname to listen on.
                                         deprecated aliases: --host, --hostname
                                                                                         [string] [default: "127.0.0.1"]
  -p, --server.port, --port              Port to listen on.
                                         deprecated aliases: --port
                                                                                                [number] [default: 8545]

Options:
  -?, --help                  Show help                                                                        [boolean]
      --version               Show version number                                                              [boolean]
  -D, --cli.detach, --detach  Run Ganache in detached mode.                               [boolean] [default: (default)]
  ```

Reply:
What if the namespace were called `process`? That seems more widely useful for future expansion, and also more precise a term